torchao/quantization/pt2e/quantizer/x86_inductor_quantizer.py

```diff
@@ -1625,3 +1625,17 @@ def _same_shape(n1: Node, n2: Node):
 
     def validate(self, model: torch.fx.GraphModule) -> None:
         pass
+
+
+# Register Inductor fusion passes
+import torch._inductor.config
+
+from torchao.quantization.pt2e.inductor_passes.x86 import (
+    _register_quantization_weight_pack_pass,
+    quant_lift_up,
+)
+from torchao.utils import TORCH_VERSION_AT_LEAST_2_8
+
+if TORCH_VERSION_AT_LEAST_2_8:
+    torch._inductor.config.pre_grad_custom_pass = quant_lift_up
```

Comment:
just confirming, is this the final API already or do we still expect more changes here?

Reply:
I think this is subject to change. Discussions are still underway. We plan to modify this part after the API is changed. Thanks.